R/add_coordinate_uncertainty.R

```diff
@@ -0,0 +1,70 @@
+#' Add coordinate uncertainty
+#'
+#' Adds a collum to the occurrences sf object with the coordinate uncertainty in meters
+#'
+#'
+#' @param occurrences A sf object with POINT geometry
+#' @param coords_uncertainty_meters a value or a vector of numeric values
+#'
+#' @return
+#' A sf object with POINT geometry with an additional column named 'coordinateUncertaintyInMeters'
+#' @export
+#'
+#' @examples
+#'
+#'
+#' library(sf)
+#' library(dplyr)
+#'
+#' set.seed(123)
+#'
+#' # Create four random points
+#' n_points <- 4
+#' xlim <- c(3841000, 3842000)
+#' ylim <- c(3110000, 3112000)
+#' observations_sf <- data.frame(
+#'   lat = runif(n_points, ylim[1], ylim[2]),
+#'     long = runif(n_points, xlim[1], xlim[2])) %>%
+#'     st_as_sf(coords = c("long", "lat"), crs = 3035)
+#'
+#'  # provide a fixed uncertainty for all points
+#'  add_coordinate_uncertainty(
+#'    observations_sf,
+#'    coordinate_uncertainty_meters = 1000
+#'    )
+#'
+#' # add variability in uncertainty. For example, using gamma distribution
+#' add_coordinate_uncertainty(
+#'   observations_sf,
+#'   coordinate_uncertainty_meters = rgamma(n_points, shape = 5, rate = 0.1)
```

Comment:
The argument has a different name in the examples than it does in the function definition, causing the examples to fail.